core/kazoo_services/src/kz_services.erl

```diff
@@ -455,14 +505,18 @@ ratedeck_name(Services) ->
 
 -spec plan_ratedeck_name(services()) -> kz_term:api_ne_binary().
 plan_ratedeck_name(Services) ->
-    Plans = kz_services_plans:foldl(fun(_BookkeeperHash, PlansList, Plans) ->
-                                            PlansList ++ Plans
-                                    end
-                                   ,[]
-                                   ,plans(Services)
-                                   ),
-    Plan = kz_services_plans:merge(Plans),
-    kz_services_plan:ratedeck_name(Plan).
+    case kz_services_plans:foldl(fun(_BookkeeperHash, PlansList, Plans) ->
+                                         PlansList ++ Plans
+                                 end
+                                ,[]
+                                ,plans(Services)
+                                )
+    of
+        [] -> 'undefined';
+        Plans ->
+            Plan = kz_services_plans:merge(Plans),
+            kz_services_plan:ratedeck_name(Plan)
+    end.
```

Comment:
Unrelated to here but Dialyzer complains:

```
  kz_services.erl:804: The call kz_services:commit_updates(Account::any(),Current::any(),Proposed::any(),[]) will never return since the success typing is ('undefined' | <<_:8,_:_*8>>,[{[any()]}],[{[any()]}],{[{atom() | <<_:8,_:_*8>>,_}]}) -> #kz_services{account_id::'undefined' | <<_:8,_:_*8>>,services_jobj::{[{_,_}]},current_services_jobj::{[{_,_}]},plans::'undefined' | kz_services_plans:plans(),account_updates::{[{_,_}]},cascade_updates::{[{_,_}]},manual_updates::{[{_,_}]},invoices::'undefined' | kz_services_invoices:invoices(),audit_log::{[{_,_}]},dirty::boolean()} and the contract is (kz_term:ne_binary(),kz_services_quantities:billables(),kz_services_quantities:billables(),kz_json:object()) -> services() 
```
I think you need to use `kz_json:new()` instead of `[]` for AuditLog; otherwise update the spec for `commit_updates/4`.

Reply:
This should be fixed in my commit